gui/build.py

```diff
@@ -79,25 +81,18 @@ def do_bundle(options):
     for dirname in ('build', 'dist'):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
-    # create a intaller
-    python_lib_path = os.path.join(options.python_path, "Lib")
-    makensis_path = os.path.join(options.nsis_path, "makensis.exe")
+    # create a installer
     gui_path = os.path.realpath('.')
     moz_path = os.path.realpath('..')
-    python_dir = os.path.join(
-        os.path.dirname(sys.executable),
-        "Scripts"
-    )
-    args = []
-    args.append('--icon=wininst/app_icon.ico')
-    args.append('--base-name=Win32GUI')
-    args.append('--include-path=%s;%s;%s\site-packages;%s'
-                % (gui_path, moz_path, python_lib_path, python_lib_path))
-    args.append('--target-name=mozregression-gui.exe')
-    args.append('--target-dir=dist')
-    args.append('mozregui/main.py')
-    call(sys.executable, os.path.join(python_dir, 'cxfreeze'), *args)
-    call(makensis_path, 'wininst.nsi', cwd='wininst')
+    python_lib_path = options.python_path
+    if IS_WIN:
+        makensis_path = os.path.join(options.nsis_path, "makensis.exe")
+        call_cx_freeze(gui_path, moz_path, python_lib_path)
```

Comment:
Since now call_cx_freeze needs to be done on the two platforms, could you move that line before the if IS_WIN ?
Also you won't need the python_lib_path variable (it will be used only once), just:
```
call_cx_freeze(gui_path, moz_path, options.python_lib_path)
```